resources/schema/spidacalc/results/clearance_rule_result.schema

```diff
@@ -0,0 +1,106 @@
+{
+  "id": "#/spidacalc/results/clearance_rule_result.schema",
+  "description": "Clearance Rule Result",
+  "type": "object",
+  "required": [
+    "clearanceCaseName",
+    "clearanceRuleName",
+    "clearanceCaseType"
```

Comment:
more than these are required... probably upperId, lowerId, allowable, actual, difference, and status at least?